src/sqlancer/mariadb/MariaDBProvider.java

```diff
@@ -126,6 +129,9 @@ public void generateDatabase(MariaDBGlobalState globalState) throws Exception {
                     query = MariaDBInsertGenerator.insert(globalState.getSchema(), globalState.getRandomly());
                     break;
                 case OPTIMIZE:
+                    if (MariaDBBugs.bug834) {
+                        break;
```

Comment:
The logic here looks and below quite complex. Can we just set `nrPerformed` to `0`?